iptables/iptables.go

```diff
@@ -53,7 +53,7 @@ func New() (*IPTables, error) {
 }
 
 // Exists checks if given rulespec in specified table/chain exists
-func (ipt *IPTables) Exists(table, chain string, rulespec...string) (bool, error) {
```

Comment:
how did this even compile?! (i.e., where are the tests blocking this from getting into master :-)